package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "egghead-ui",
-  "version": "2.18.0",
+  "version": "2.19.0",
```

Comment:
Did this using the new `yarn bump` (to bump here and create a new git tag), which triggers Travis CI to release a new version on npm once merged to master.